README.md

```diff
@@ -56,8 +56,6 @@ If you don't see a version number, then make sure Git is on your `PATH`.
 
 ## Step 4: Clone this repository
```

Comment:
This section is empty now. Remove it or add link to appropriate page.

Reply:
Do we want the public repo pointing to the private site?

Reply:
I've added a sentence explaining to follow the instructions in the first project.